src/app/issuer/components/issuer-list/issuer-list.component.html

```diff
@@ -136,7 +136,7 @@ <h1 class="tw-font-bold tw-text-purple tw-text-3xl tw-mb-8">
 								class="tw-text-oebblack tw-pb-2 tw-font-semibold md:tw-text-[20px] md:tw-leading-[24.4px] tw-text-[14px] tw-leading-[19.6px]"
 							>
 								{{ 'Issuer.yourRole' | translate }}
-								{{ issuer.currentUserStaffMember?.roleInfo.label }}
+								{{ issuer.currentUserStaffMember?.roleInfo.label | translate }}
```

Comment:
What will happen if this is ever null or undefined? Does the translate pipe handle that gracefully?